# First IMU in 'imu_mapping' will be used
# as the base for the visualization node

/**:
  ros__parameters:
    header_rate: 100.0
    imu_prefix: "00B483"
    imu_mapping:
      [
        "A1", "torso_imu",
        "84", "pelvis_imu",
        "9D", "femur_l_imu",
        "9C", "femur_r_imu",
        "A4", "tibia_l_imu",
        "A0", "tibia_r_imu",
        "83", "calcn_l_imu",
        "9F", "calcn_r_imu"
      ]



# A dictionary would be more practical,
# but cannot be used in ROS2 parameters.
# Make sure to adjust the TF offsets in
# the xsens_mtw_visualization node when
# using a different setup.
#
# EXAMPLE:
#
# imu_mapping:
#   [
#   "A1", "torso_imu",
#   "84", "pelvis_imu",
#   "9D", "femur_l_imu",
#   "9C", "femur_r_imu",
#   "A4", "tibia_l_imu",
#   "A0", "tibia_r_imu",
#   "83", "calcn_l_imu",
#   "9F", "calcn_r_imu"
#   ]


# Using wildcards:
# https://docs.ros.org/en/humble/Tutorials/Intermediate/Launch/Using-ROS2-Launch-For-Large-Projects.html#using-wildcards-in-yaml-files
